.github/workflows/docs-link-check.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           # Set GITHUB_TOKEN to avoid github rate limits on URL checks
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          cd docs
```

Comment:
What was the reason for this change? Link checker should be running on all markdown files in the repo, not just those inside docs/ folder. There are plenty of READMEs and examples throughout.

If the goal was to exclude some specific dir, we can do that if needed.